docs/content/stable/yugabyte-platform/install-yugabyte-platform/prepare-on-prem-nodes.md

```diff
@@ -13,6 +13,34 @@ type: docs
 
 For on-premises deployments of YugabyteDB universes, you need to import nodes that can be managed by YugabyteDB Anywhere.
 
+## Prerequisites
+
+Using YugabyteDB Anywhere (YBA), you can deploy YugabyteDB universes on nodes with the following architectures and operating systems.
+
+### Supported CPU architectures
+
+YBA supports deploying YugabyteDB on both x86 and ARM (aarch64) architecture-based hardware.
+
+Note that support for ARM architectures is unavailable for airgapped setups, because YBA ARM support for [AWS Graviton](https://aws.amazon.com/ec2/graviton/) requires Internet connectivity.
+
+### Supported operating systems
+
+YBA supports deploying YugabyteDB on a variety of [operating systems](../../../reference/configuration/operating-systems/#yugabytedb-anywhere). YBA additionally supports deploying YugabyteDB on Red Hat Enterprise Linux 8. AlmaLinux OS 9 is used by default.
+
+#### Requirements for all OSes
+
+Python 3 is required. If you're using YugabyteDB Anywhere to provision nodes in public clouds, be sure the custom AMI you plan to use has Python 3 installed.
```

Comment:
Can we also put something about how if SELinux is enabled we will also need the corresponding selinux, policycoreutils-python, and python3-libselinux packages. Also their python3 should be accessible by invoking just `python3` (so should be in their path ig, not sure the right wording for that).